core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -3305,14 +3311,29 @@ Failure to provide the required information to the Mediator or Arbitrator will r
 In all dispute cases, the USPMO sender bears 100% of the burden of responsibility in providing evidence/proof to the Mediator or Arbitrator.\n\n\
 If you do not understand these requirements, do not trade using USPMO on Bisq.
 
+payment.cashByMail.info=Trading using cash-by-mail (CBM) on Bisq requires that you understand the following:\n\
+  \n\
+  ● BTC buyer should package cash in a tamper-evident cash bag.\n\
+  ● BTC buyer should film or take high-resolution photos of the cash packaging process with the address & tracking number already affixed to packaging.\n\
+  ● BTC buyer should send the cash package to the BTC seller with Delivery Confirmation and appropriate Insurance.\n\
+  ● BTC seller should film the opening of the package, making sure that the tracking number provided by the sender is visible in the video.\n\
+  ● Offer maker must state any special terms or conditions in the 'Additional Information' field of the payment account.\n\
+  ● Offer taker agrees to the offer maker's terms and conditions by taking the offer.\n\
+  ● MEDIATORS CANNOT MAKE JUDGEMENTS IN CBM DISPUTES. But they can help with smaller problems such as technical difficulties not related to scam attempts.\n\
+  \n\
+  To be sure you fully understand the requirements of cash-by-mail trades, please see: [HYPERLINK:https://bisq.wiki/Cash_by_Mail]\n\
+  \n\
+  If you do not understand these requirements, do not trade using CBM on Bisq.
+
+payment.cashByMail.contact=Contact info
+payment.cashByMail.contact.prompt=Your contact information such as name or nym
 payment.f2f.contact=Contact info
 payment.f2f.contact.prompt=How would you like to be contacted by the trading peer? (email address, phone number,...)
 payment.f2f.city=City for 'Face to face' meeting
 payment.f2f.city.prompt=The city will be displayed with the offer
-payment.f2f.optionalExtra=Optional additional information
-payment.f2f.extra=Additional information
-
-payment.f2f.extra.prompt=The maker can define 'terms and conditions' or add a public contact information. It will be displayed with the offer.
+payment.shared.optionalExtra=Optional additional information
+payment.shared.extraInfo=Additional information
+payment.shared.extraInfo.prompt=The maker can define 'terms and conditions' or add a public contact information. It will be displayed with the offer.
```

Comment:
```suggestion
payment.shared.extraInfo.prompt=Define any special terms, conditions, or details you would like to be displayed with your offers for this payment account (users will see this info before accepting offers).
```
I know this string isn't changed by this PR per se, so I understand if it's preferred to implement the suggestion on another PR.

See bottom of screenshot below for context on where this string shows.

![Screenshot from 2021-01-20 11-23-55](https://user-images.githubusercontent.com/735155/105204533-2f9e4900-5b3c-11eb-8aab-aac2363a536d.png)